sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/TelemetryPartB.cs

```diff
@@ -123,6 +129,53 @@ private static void AddPropertiesToTelemetry(IDictionary<string, string> destina
             }
         }
 
+        private static void AddActivityLinksToPartCTags(IEnumerable<ActivityLink> links, ref AzMonList PartCTags)
+        {
+            string msLinks = "_MS.links";
+            // max number of links that can fit in this json formatted string is 107. it is based on assumption that traceid and spanid will be of fixed length.
+            // Keeping max at 100 for now.
+            // todo: define this as constant.
+            int maxLinksAllowed =100;
```

Comment:
I kept it 100 here for now. Max links that can fit into 8192 character limit are 107. Open for suggestions/comments.